Alloy/CVPixelBuffer+MTLTexture.swift

```diff
@@ -10,12 +10,22 @@ import Metal
 import CoreVideo.CVPixelBuffer
 
 public extension CVPixelBuffer {
-    func metalTexture(using cache: CVMetalTextureCache, pixelFormat: MTLPixelFormat, planeIndex: Int = 0) -> MTLTexture? {
+    func texture(using cache: CVMetalTextureCache,
```

Comment:
does ditching `metal` word really helps here?

Reply:
less to type, more calories left

Reply:
I think it is more explicit since cache is named `CV**Metal**TextureCache `